plugins/org.locationtech.udig.ui/src/org/locationtech/udig/ui/FeatureTypeEditor.java

```diff
@@ -358,8 +390,17 @@ public SimpleFeatureType createDefaultFeatureType() {
         builder = new SimpleFeatureTypeBuilder();
         builder.setName(Messages.FeatureTypeEditor_newFeatureTypeName); 
         builder.setCRS(getDefaultCRS());
-        builder.length(MAX_ATTRIBUTE_LENGTH).add(Messages.FeatureTypeEditor_defaultNameAttributeName, String.class); 
-        builder.add(Messages.FeatureTypeEditor_defaultGeometryName, LineString.class); 
+        //no need to provide a default length since length can be specified via the UI
+        //builder.length(MAX_ATTRIBUTE_LENGTH).add(Messages.FeatureTypeEditor_defaultNameAttributeName, String.class); 
```

Comment:
Is the constant MAX_ATTRIBUTE_LENGTH referenced somewhere else or coult it be deleted? IMHO the comment on the constant indicates that it is a specific value for a specific database which might not make any sense for other storage types (file/other databases)

Reply:
No it is not used. Initialy it was used for setting a default length for the lenght of an attribute but this become obsolete since the new column for etting lenght was intriduced